tests/fuzzer/readme.md

```diff
@@ -0,0 +1,35 @@
+## libFuzzer
```

Comment:
There's 2 other fuzzers in this dir (for parser and verifier), please mention them in this readme, and maybe show how ti run those also?